apps/passport-client/src/dispatch.ts

```diff
@@ -569,7 +572,13 @@ async function handlePasswordChangeOnOtherTab(update: ZuUpdate) {
   const encryptionKey = loadEncryptionKey();
   return update({
     self,
-    encryptionKey
+    encryptionKey,
+
+    // Extra download helps to get our in-memory sync state up-to-date faster.
+    // The password change on the other tab is an upload of a new revision so
+    // a download is necessary.  Otherwise loading our new self object above
+    // will make this tab think it needs to upload and cause a conflict.
+    extraDownloadRequested: true
```

Comment:
You know, I've forgotten what exactly `extraDownloadRequested` does. I think it's probably worth commenting its purpose at the place it's defined, i.e. in `state.ts`

Reply:
I'm not against that in the abstract, though my thought when adding it (following the pre-existing pattern at the time) was that the single place it's used (inside of sync()) is self-documenting with a big comment.  I can try documenting all of the sync-state-machine fields.